### PR TITLE
Get server URL from vip configmap

### PR DESCRIPTION
**Pull request by yaocw2020**
Get server URL from vip configmap

issue: https://github.com/harvester/harvester/issues/1019

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM

**Comment by guangbochen** [comment]
@yaocw2020 can we add server URL address validation, thanks

File: pkg/api/kubeconfig/generate_handler.go [hunk 1]
```diff
@@ -36,9 +41,11 @@ type GenerateHandler struct {
 	roleBindingCache  ctlrbacv1.RoleBindingCache
 	secretCache       ctlcorev1.SecretCache
 	secretClient      ctlcorev1.SecretClient
+	configMapCache    ctlcorev1.ConfigMapCache
+	namespace         string
 }
 
-func NewGenerateHandler(scaled *config.Scaled) *GenerateHandler {
+func NewGenerateHandler(scaled *config.Scaled, option *config.Options) *GenerateHandler {
```
Comment (guangbochen) [inline pkg/api/kubeconfig/generate_handler.go]: don't need to pass the option as pointer, pass value is sufficient enough

**Comment by gitlawr** [comment]
Why don't we use the kubernetes core API?

**Comment by yaocw2020** [comment]
> Why don't we use the kubernetes core API?

Agree, I have deleted the deleteKubeconfig API.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**